tutorials/intermediate/blitting.py

```diff
@@ -0,0 +1,189 @@
+"""
+=================
+Blitting Tutorial
```

Comment:
```suggestion
Blitting tutorial
```

Speaking of style guides in the call :smirk: 
https://matplotlib.org/devdocs/devel/documenting_mpl.html#section-name-formatting

Reply:
:sheep: